packages/affiliates/test/mocha/affiliates-utils.js

```diff
@@ -0,0 +1,115 @@
+const { assert } = require("chai");
```

Comment:
unit tests for the utility file which needs to be expanded on, though most of the functions are tested in production